src/app/containers/Article/index.jsx

```diff
@@ -11,49 +11,87 @@ import ArticleMain from '../ArticleMain';
 import ErrorMain from '../ErrorMain';
 import nodeLogger from '../../helpers/logger.node';
 import ConsentBanner from '../ConsentBanner';
+import { GhostWrapper, GridItemConstrainedMedium } from '../../lib/styledGrid';
 
 const logger = nodeLogger(__filename);
 
-/*
-  [1] This handles async data fetching, and a 'loading state', which we should look to handle more intelligently.
-*/
+const ArticleWrapper = ({ bbcOrigin, children, isAmp, service }) => (
```

Comment:
@sareh Do you think Article wrapper could be transitioned into a more generic PageWrapper?

This way we could have a single HOC that is used to wrap our page types (Article and Home Page for now) and lift the loading state into this HOC or higher. Then the same Page wrapper, loading and error handling functionality could be used through our page types.

In this case the children would be, the article container, the homepage container and the error pages container.